README.md

```diff
@@ -37,8 +37,41 @@ And these database configuration settings:
 
 ## Local Development
 
-Requires docker.
+### Local Application Development
```

Comment:
Not going to block changes since I'm not on this work cycle, but I wonder if you would consider changing a couple labels in this change. (I'm not clear on how docker-based work is not local, I guess.) In specific, would you consider changing the `Local Application Development` subheader to `Local Development (Without Docker)`, and `Local E2E Development` to `Local Development (Using Docker)`, or some such?

Reply:
The docker section is a heading under local development (so there are sections under local development, one for w/ Docker, and one w/o).  This was not to indicate that working w/ Docker was not local, but that there was two modalities.

That being said, happy to make this change.

Reply:
> yo dawg, i herd u like local development

Thx, @jkeck!